docs/usage/interacting-with-node.md

```diff
@@ -11,6 +11,8 @@ Once the Centrifuge node is up and running you are able to start submitting invo
 Confirm that your Centrifuge Node is up und running, for example with a PING (See section [Post install verification](/docs/getting-started/ping)). If it is not running, run the Centrifuge Node using the `config.yaml` file you created:
```

Comment:
> If it is not running, run the Centrifuge Node using the `config.yaml` file you created. 

Perhaps add: "Recall that you may also need to add your Ethereum key and password as environment variables in advance."